gdal/swig/python/osgeo/utils/gdal2xyz.py

```diff
@@ -37,15 +37,18 @@
 import numpy as np
 
 from osgeo import gdal
-from osgeo.utils.auxiliary.base import num
+from osgeo.utils.auxiliary.base import num, PathLike
+from osgeo.utils.auxiliary.util import PathOrDS
 from osgeo.utils.auxiliary.numpy_util import GDALTypeCodeAndNumericTypeCodeFromDataSet
 
 
 def Usage():
-    print('Usage: gdal2xyz.py [-skip factor] [-srcwin xoff yoff width height]')
-    print('                   [-band b] [--allBands] [-csv]')
-    print('                   [--skipNoData] [-noDataValue value]')
-    print('                   srcfile [dstfile]')
+    print('Usage: gdal2xyz.py [-help]'
```

Comment:
Question: Help is not explicitly defined, I assume it will just trigger the main else case in parsing? 
Should we add it explicitly?

Reply:
Right, I've added explicitly it as you suggested.